doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

```diff
@@ -36,6 +36,34 @@ Libraries
 
 This section describes the changes related to libraries.
 
+Google Fast Pair
+----------------
+
+.. toggle::
+
+   For applications and samples using the :ref:`bt_fast_pair_readme` library:
+
+   * If you use sysbuild for generating a hex file with the Fast Pair provisioning data, you must align your application with the new approach for passing the provisioning parameters (the Model ID and the Anti-Spoofing Private Key).
+     The ``FP_MODEL_ID`` and ``FP_ANTI_SPOOFING_KEY`` CMake variables are replaced by the corresponding ``SB_CONFIG_BT_FAST_PAIR_MODEL_ID`` and ``SB_CONFIG_BT_FAST_PAIR_ANTI_SPOOFING_PRIVATE_KEY`` Kconfig options that are defined at the sysbuild level.
+     The following additional build parameters for Fast Pair are no longer valid:
+
+     ``-DFP_MODEL_ID=0xFFFFFF -DFP_ANTI_SPOOFING_KEY=AbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbA=``
+
+     And must be replaced with the new sysbuild Kconfig options.
+     You can provide them as additional build parameters to the build command as follows:
+
+     ``-DSB_CONFIG_BT_FAST_PAIR_MODEL_ID=0xFFFFFF -DSB_CONFIG_BT_FAST_PAIR_ANTI_SPOOFING_PRIVATE_KEY=\"AbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbA=\"``
+
+     You can replace this exemplary method with any other configuration method that is supported by sysbuild.
+
+     .. note::
+        To avoid build failures, you must surround the string value for the Anti-Spoofing Private Key parameter with the ``\"`` characters instead of the typical ``"`` character.
```

Comment:
this might work for linux but probably doesn't work for windows or vice-versa

Reply:
thanks, tested using Windows and found the syntax that works for Windows